src/server/index.spec.tsx

```diff
@@ -106,5 +108,102 @@ describe(__filename, () => {
         expect(response.text).toContain(expectedHTML);
       });
     });
+
+    describe('NODE_ENV=development', () => {
+      let proxy: express.Application;
+      let proxyServer: http.Server;
+
+      const devEnv = {
+        NODE_ENV: 'development',
+        REACT_APP_CRA_PORT: 60123,
+      };
+
+      beforeEach(() => {
+        proxy = express();
+        proxyServer = proxy.listen(devEnv.REACT_APP_CRA_PORT);
```

Comment:
This needs some kind of error handling, otherwise proxy errors show up as unhandled promises (the tests still fail with expected errors, though).

<img width="464" alt="screenshot 2019-02-13 14 58 59" src="https://user-images.githubusercontent.com/55398/52743652-17ad0a80-2fa0-11e9-8bfb-0f1ede065a1e.png">

This is the best I can think of:

```diff
diff --git a/src/server/index.spec.tsx b/src/server/index.spec.tsx
index 1c94e11..8a96302 100644
--- a/src/server/index.spec.tsx
+++ b/src/server/index.spec.tsx
@@ -121,6 +121,9 @@ describe(__filename, () => {
       beforeEach(() => {
         proxy = express();
         proxyServer = proxy.listen(devEnv.REACT_APP_CRA_PORT);
+        proxyServer.on('error', (error) => {
+          console.error('proxy error', error);
+        });
       });
 
       afterEach(() => {
```

After that, I don't see unhandled promise rejections.

Reply:
:+1: